From 8cb9514b8637b6756c802cc6de0d68cfa626bde1 Mon Sep 17 00:00:00 2001
From: mbrinkma <mbrinkma@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Thu, 28 Jan 2021 18:20:02 +0000
Subject: [PATCH 405/462] MANTIS 0059803 Toronto T1: Repeat barcode-scan at
 insertion-area if barcode could not be detected in 1st try - tests,
 test-framework-improvements

git-svn-id: svn://localhost/SelfServiceBuild/trunk@739 2eb42a87-3827-0410-b767-dc15055982e4
---
 .../scripts/java8/gradle/wrapper/gradle-wrapper.properties      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SelfServiceBuild/Build/scripts/java8/gradle/wrapper/gradle-wrapper.properties b/SelfServiceBuild/Build/scripts/java8/gradle/wrapper/gradle-wrapper.properties
index 7c4388a..1c4bcc2 100644
--- a/SelfServiceBuild/Build/scripts/java8/gradle/wrapper/gradle-wrapper.properties
+++ b/SelfServiceBuild/Build/scripts/java8/gradle/wrapper/gradle-wrapper.properties
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.1-all.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-- 
2.41.0.windows.1

